plugins/catalog/src/components/AboutCard/fields.tsx

```diff
@@ -0,0 +1,287 @@
+        label="Owner"
+        value="No Owner"
+        gridSizes={{ xs: 12, sm: 6, lg: 4 }}
+      >
+        {ownedByRelations.length > 0 && (
+          <EntityRefLinks entityRefs={ownedByRelations} defaultKind="group" />
+        )}
+      </AboutField>
+    );
+  },
+  Domain() {
+    const { entity } = useEntity();
+    const isSystem = entity.kind.toLocaleLowerCase('en-US') === 'system';
+    const partOfDomainRelations = getEntityRelations(entity, RELATION_PART_OF, {
+      kind: 'domain',
+    });
+    return (
+      <>
+        {(isSystem || partOfDomainRelations.length > 0) && (
+          <AboutField
+            label="Domain"
+            value="No Domain"
+            gridSizes={{ xs: 12, sm: 6, lg: 4 }}
+          >
+            {partOfDomainRelations.length > 0 && (
+              <EntityRefLinks
+                entityRefs={partOfDomainRelations}
+                defaultKind="domain"
+              />
+            )}
+          </AboutField>
+        )}
+      </>
+    );
+  },
+  System() {
+    const { entity } = useEntity();
+    const validEntities = ['api', 'component', 'resource'];
+    const isValidEntity = validEntities.includes(
+      entity.kind.toLocaleLowerCase('en-US'),
+    );
+    const partOfSystemRelations = getEntityRelations(entity, RELATION_PART_OF, {
+      kind: 'system',
+    });
+    return (
+      <>
+        {(isValidEntity || partOfSystemRelations.length > 0) && (
+          <AboutField
+            label="System"
+            value="No System"
+            gridSizes={{ xs: 12, sm: 6, lg: 4 }}
+          >
+            {partOfSystemRelations.length > 0 && (
+              <EntityRefLinks
+                entityRefs={partOfSystemRelations}
+                defaultKind="system"
+              />
+            )}
+          </AboutField>
+        )}
+      </>
+    );
+  },
+  Parent() {
+    const { entity } = useEntity();
+    const isComponent = entity.kind.toLocaleLowerCase('en-US') === 'component';
+    const partOfComponentRelations = getEntityRelations(
+      entity,
+      RELATION_PART_OF,
+      {
+        kind: 'component',
+      },
+    );
+    return (
+      <>
+        {isComponent && partOfComponentRelations.length > 0 && (
+          <AboutField
+            label="Parent Component"
+            value="No Parent Component"
+            gridSizes={{ xs: 12, sm: 6, lg: 4 }}
+          >
+            <EntityRefLinks
+              entityRefs={partOfComponentRelations}
+              defaultKind="component"
+            />
+          </AboutField>
+        )}
+      </>
+    );
+  },
+  Type() {
+    const { entity } = useEntity();
+    const validEntities = [
+      'api',
+      'component',
+      'resource',
+      'template',
+      'group',
+      'location',
+    ];
+    const isValidEntity = validEntities.includes(
+      entity.kind.toLocaleLowerCase('en-US'),
+    );
+    return (
+      <>
+        {(isValidEntity || typeof entity?.spec?.type === 'string') && (
```

Comment:
Same here, just your typeof is fine. Could also break that one out if you like:

```ts
const type = entity?.spec?.type;
if (typeof type === 'string') {
  // inside here, typescript is smart enough to know that the variable is of string type without casting
}
```

Reply:
Actually, TS is smart enough to know that the variable is a string just by having `typeof entity?.spec?.type === 'string'`, the type casting after is redundant.